src/H5public.h

```diff
@@ -60,7 +60,7 @@
 #define MPICH_SKIP_MPICXX 1
 #endif
 #ifndef OMPI_SKIP_MPICXX
-#define OMPI_SKIP_MPICXX  1
+#define OMPI_SKIP_MPICXX 1
```

Comment:
Looks like this change is getting pulled into different PRs?

Reply:
yes, we let one get by and now it has snowballed!